src/server/lib/example.js

```diff
@@ -6,3 +6,21 @@ exports.pluralize = function (count, singular, plural) {
   return `${count} ${plural}`;
 };
 
+
+exports.pluralize_v2 = function (count, singular, plural) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'pluralize_v2' is not in camel case. (camelcase)](https://app.codacy.com/app/Codacy-ACME/nodeproject/pullRequest?prid=4310063)